src/index.md

```diff
@@ -24,7 +24,7 @@ Known issues left for future versions include:
 
 * adaptive mesh topology (this could be supported by defining a `time_concatenation` attribute for a time-series of mesh topologies)
 * higher order element data; for an idea how such data could be stored see this other [proposal](https://publicwiki.deltares.nl/display/NETCDF/Finite+Element+based+CF+proposal+for+Unstructured+Grid+data+model).
-* subgrid data; the NetCDF pages by the Bundesanstalt für Wasserbau (BAW) contain some proposals on this topic ([see their pages (in German)](http://www.baw.de/methoden/index.php5/NetCDF)).
+* subgrid data; the NetCDF pages by the Bundesanstalt für Wasserbau (BAW) contain some proposals on this topic ([see their pages (in German)](https://wiki.baw.de/de/index.php?title=NetCDF)).
```

Comment:
There is a english version at:
https://wiki.baw.de/en/index.php/NetCDF

perhaps we should link directly to that url?

Reply:
Good spot! I am not on the UGRID team/committee so it is not strictly my say/decision to make, but my opinion is that it would be much better to link to the English language version, to be consistent with the rest of the conventions document.

Since I guess you agree given the suggestion, I will update the link to that now, in a new commit, but can always revert it if someone prefers otherwise. Thanks.

Reply:
Thanks @ChrisBarker-NOAA for your timely response and suggestion. I've added a new commit to update the link to the English version: see [5ca24d3](https://github.com/ugrid-conventions/ugrid-conventions/pull/71/commits/5ca24d335d0d0357eb174fc1eed6b2a766b37c4a). Let me know if there's anything else I should change or add/remove.